pkg/controller/configmap/config.go

```diff
@@ -43,7 +43,7 @@ func FillDefaults(configmap *corev1.ConfigMap, spec *configv1.NetworkSpec) error
 	appendErrorIfNotNil(&errs, fillDefault(cfg, "coe", "enable_snat", "True", false))
 	appendErrorIfNotNil(&errs, fillDefault(cfg, "ha", "enable", "True", false))
 	appendErrorIfNotNil(&errs, fillDefault(cfg, "k8s", "process_oc_network", "False", true))
-    // For Openshift add a 3-seconds agent delay by default
+	// For Openshift add a 3-seconds agent delay by default
```

Comment:
thanks! I think I authored that patch with the editor with the python settings...